PublicGitArchive/doc/dataset_analysis.md

```diff
@@ -0,0 +1,71 @@
+# Description of the current dataset
+
+## 1. GHTorrent: selecting the list of URLs
+
+From GHTorrent’s MySQL dump, dated from 01/01/2018.
+By querying repos with at least 50 stargazers
+
+Output: list of 190,051 URLs with duplicates  
+List of unique URLs: 187,352
+
+## 2. Collection with Borges
+
+Output of Borges:  
+
+183,015 **repositories**  
+4,338 are **indefinitely pending** for several reasons :  
+* private repos  
+* not existing anymore  
+* borges is not able to process them, maybe a bug
+
+## 3. Dataset stats in HDFS
+
+Total **size** : 2.7 TB  
+Number of **siva files** : 278,923
+
+
+## 4. Latest CSV file : `dataset_stats.ipynb`
+
+bug?: row nº 171531 contains only the titles of the columns, see `dataset_stats.ipynb`. It is dropped to compute the rest of the stats.
```

Comment:
Because we concatenated two index files together, right @erizocosmico? In any case, we need to remove this line, update the index file and remove this remark.